components/partials/activity/derivatives/trades.vue

```diff
@@ -3,27 +3,35 @@
     <v-card-table-wrap>
       <template #actions>
         <div
-          class="col-span-12 lg:col-span-6 grid grid-cols-2 sm:grid-cols-3 gap-4 w-full"
+          class="col-span-12 lg:col-span-6 grid grid-cols-5 sm:grid-cols-3 gap-4 w-full"
         >
           <v-search
             dense
-            class="col-span-2 sm:col-span-1"
+            class="col-span-3 sm:col-span-1"
             :placeholder="$t('trade.filter')"
             :search="search"
             data-cy="universal-table-filter-by-asset-input"
             @searched="handleInputOnSearch"
           />
 
+          <div
+            class="col-span-2 flex items-center bg-gray-900 rounded-full text-gray-200 py-3 px-6 text-xs cursor-pointer sm:hidden shadow-sm"
+            @click="openMobileFilterModal"
+          >
+            <VIconFilter class="min-w-4 mr-2" />
```

Comment:
Nitpicking but our convention so far has been ```<v-icon-filter />```, I believe we're now moving to this convention: ```<IconFilter />``` for files within our codebase and ```<VSomeComponent />``` for third party files right?